.eslintrc.js

```diff
@@ -3,14 +3,21 @@
 module.exports = {
   root: true,
   extends: ['sentry-app/strict'],
+
+  parserOptions: {
+    project: './tsconfig.json',
+  },
```

Comment:
eh i'm not sure its worth it to start using the tsc backed parser for a bunch of warnings

Reply:
thats fair. i think ideally these aren't enabled on every PR, because folks will just get desensitized to it. Instead I'd like a way to generate the list on-demand, but also have a number reported every week or something so we can track progress.